### PR TITLE
Component For "Add UI/UX message about new Po.et URLs"

### PR DESCRIPTION
**Pull request by jwicks31**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

## Description of Changes
Adds an `Announcement Component` to the Dashboard page. 

- Hidden behind `announcement` feature toggle.
- Currently Lorem ipsum text and a link to home page.
- Visit `/dashboard?ft=announcement` to test.

Part of [#207](https://github.com/poetapp/random/issues/207) in Random Repo

### Warning

Merging to `master` automatically triggers a deploy in production.

### CODE REVIEW

File: src/components/atoms/Announcement/Announcement.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import 'colors';
+
+.Announcement {
+  width: 100%;
+  margin-bottom: -1.5rem;
```
Comment (wzalazar) [inline src/components/atoms/Announcement/Announcement.scss]: Can you avoid negative margins?

File: src/components/molecules/PanelOptions/PanelOptions.tsx [hunk 2]
```diff
@@ -1,4 +1,9 @@
+import { isActiveFeatureName } from '@paralleldrive/feature-toggles'
+import { Feature } from '@paralleldrive/react-feature-toggles'
+
+import { Announcement } from 'components/atoms/Announcement/Announcement'
 import { BoxButton } from 'components/atoms/BoxButton/BoxButton'
+import { FeatureName } from 'config/features'
 import { Images } from 'images/Images'
 import * as React from 'react'
```
Comment (wzalazar) [inline src/components/molecules/PanelOptions/PanelOptions.tsx]: Can you move to the top of the file the import of React?

File: src/components/styles/colors.scss [hunk 3]
```diff
@@ -29,3 +29,4 @@ $nero: #222222;
 $padua: #B0E9CA;
 $eucalyptus: #268B54;
 $blue-frost: #4085DE;
+$announcement: rgba(0, 255, 0, .25)
```
Comment (wzalazar) [inline src/components/styles/colors.scss]: missing semicolon?

File: src/components/atoms/Announcement/Announcement.scss (hunk 1 above)
Comment (jwicks31) [reply src/components/atoms/Announcement/Announcement.scss]: @wzalazar OK made the changes. Thank you!

**Pull request merged**